docs/content/direct/examples.md

```diff
@@ -70,12 +79,14 @@ manager which connects to the `wds1` front-end and the `imbs1` OCM control plane
 1. Run the OCM based transport controller in a pod.  
 **NOTE**: This is work in progress, in the future the controller will be deployed through a Helm chart.
 
-   Run [transport deployment script](../../../scripts/deploy-transport-controller.sh).  
-   This script expects to get three arguments - (1) wds name; (2) imbs name; and (3) transport controller image.  
+   Run [transport deployment script](../../../scripts/deploy-transport-controller.sh), as follows.
+   This script requires that the user's current kubeconfig context be for the kubeflex hosting cluster.
+   This script expects to get two or three arguments - (1) wds name; (2) imbs name; and (3) transport controller image.  
    While the first and second arguments are mandatory, the third one is optional.
    The transport controller image argument can be specified to a specific image, or, if omitted, it defaults to the OCM transport plugin release that preceded the KubeStellar release being used.
-   For example, one can deploy transport controller using the following command:
+   For example, one can deploy transport controller using the following commands:
    ```shell
+   kubectl config use-context kind-kubeflex
```

Comment:
The above will work for kind deployment but not OCP, where the kubeflex context has a different name.

Perhaps a more generic approach is to use `kflex ctx`

Reply:
Good point. Done.